maturity-rating/index.md

```diff
@@ -0,0 +1,15 @@
+## What is a maturity rating?
+
+Before you unhide your game and show it to the world, you have to give it a maturity rating. We don't mind having games with mature content on the site as long as they are rated appropriately.
+
+Assign your game a maturity rating, then take a few moments to go through the list and select anything that applies to your game. Once you've done that, consider if your maturity rating is still appropriate. When you're satisfied that your choice is an accurate reflection of the game's content, go ahead and set the maturity rating. And remember, you can come back and make changes at any time.
```

Comment:
instead of this, we should explain what a maturity rating is and why it's important. As is, this is walking through them on how to set their maturity rating without showing them how.